doctrine/pdo_session_storage.rst

```diff
@@ -70,7 +70,8 @@ To use it, first register a new handler service:
 
     Configure the database credentials
     :doc:`using environment variables in the config file </configuration/environment_variables>`
-    to make your application more secure.
+    or :doc:`using secrets in the config file </configuration/secrets>`to make
```

Comment:
`or :doc:"using secrets in the config file </configuration/secrets>"to m` -> `or :doc:"using secrets in the config file </configuration/secrets>" to m` (missing space)